### DISCUSSION TITLE
Relation managers - disable clickable rows

### DISCUSSION THREAD

**Discussion by moleculech** [filamentphp/filament#4263] [Help] [answered]
Hi

For tables I can **disable** clickable rows using the `getTableRecordUrlUsing` function.
https://filamentphp.com/docs/2.x/tables/getting-started#record-urls-clickable-rows

The function exists in `Filament\Resources\RelationManagers` but overriding and returning null has not the intended effect.

Thanks for the help

**Comment by ManojKiranA** [comment, marked as answer]
Can you try this

```
protected function getTableRecordActionUsing(): Closure
    {
        return function (Model $record) {
            return null;
        };
    }

protected function getTableRecordUrlUsing(): Closure
    {
        return function (Model $record) {
            return null;
        };
    }
```

**Reply by danharrin** [reply]
Correct, because relation managers dont use row URLs. You don't need the `getTableRecordUrlUsing()` override.

**Reply by moleculech** [reply]
Thanks!
